usr/src/uts/common/fs/zfs/txg.c

```diff
@@ -526,10 +529,13 @@ txg_sync_thread(dsl_pool_t *dp)
 static void
 txg_quiesce_thread(dsl_pool_t *dp)
 {
+	char name[TXG_THREAD_MAXNAMELEN];
+	spa_t *spa = dp->dp_spa;
 	tx_state_t *tx = &dp->dp_tx;
 	callb_cpr_t cpr;
 
-	txg_thread_enter(tx, &cpr);
+	snprintf(name, TXG_THREAD_MAXNAMELEN, "txg_quiesce %s", spa_name(spa));
```

Comment:
for future-proof-ness, I prefer to use sizeof (name) than TXG_THREAD_MAXNAMELEN.